internal/olm/client/client.go

```diff
@@ -173,6 +173,9 @@ func (c Client) DoRolloutWait(ctx context.Context, key types.NamespacedName) err
 					log.Printf("  Waiting for Deployment %q to rollout: %d of %d updated replicas are available",
```

Comment:
It is missing the fragment. could you please add ?

Reply:
@clamoriniere1A I reverted the added additional print to debug CI failures. And also added the changelog fragment.